packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -2423,9 +2461,21 @@ const completeBoundaryWithStylesScript1Partial = stringToPrecomputedChunk(
   '$RR("',
 );
 const completeBoundaryScript2 = stringToPrecomputedChunk('","');
-const completeBoundaryScript2a = stringToPrecomputedChunk('",');
-const completeBoundaryScript3 = stringToPrecomputedChunk('"');
-const completeBoundaryScript4 = stringToPrecomputedChunk(')</script>');
+const completeBoundaryScript3a = stringToPrecomputedChunk('",');
+const completeBoundaryScript3b = stringToPrecomputedChunk('"');
+const completeBoundaryScriptEnd = stringToPrecomputedChunk(')</script>');
+
+const completeBoundaryData1 = stringToPrecomputedChunk(
+  '<template data-rci="" data-bid="',
+);
+const completeBoundaryWithStylesData1 = stringToPrecomputedChunk(
+  '<template data-rri="" data-bid="',
+);
+const completeBoundaryData2 = stringToPrecomputedChunk('" data-sid="');
+const completeBoundaryData3aStart = stringToPrecomputedChunk('" data-sty=\'');
```

Comment:
I don't think it matters much but I don't think this needs to be an escaped single quote and can still just be a regular double quote. When you write the attributes you end up escaping them for HTML which takes the double quotes generated by JSON.stringify and converts them to &quot;

Is there a security implication I am not considering that makes the single quote functionally better than double quote here?

Reply:
Thanks for catching this! I think it may have been left over from a previous version